server/server.go

```diff
@@ -380,3 +513,24 @@ func openBrowser(url string) error {
 		return fmt.Errorf("unsupported platform")
 	}
 }
+
+// GetFreePort searches for free port.
+func GetFreePort(port, limit int) int {
+	iteration := 0
```

Comment:
Iteration must start from user available ports - https://superuser.com/questions/1631280/what-exactly-are-user-ports and it must be checked. Otherwise `sudo` may be required, which is not intended.